### ISSUE TITLE
Use actual (observed) mag when available after N observations

### ISSUE THREAD

**Issue by taldcroft** [sot/starcheck#118]
Where N is some number like 5-ish.

**Comment by jeanconn** [comment]
I've forgotten; was this just for the probabilities?  Or some other way of displaying this content?

**Comment by taldcroft** [comment]
This means replacing the AGASC MAG_ACA with the observed value.  At least this seems like the simplest implementation, though I haven't looked at the code.

**Comment by jeanconn** [comment]
Would that be appropriate in, say, the starcheck catalog table, where we
expect to see the AGASC magnitude?  ​

**Comment by taldcroft** [comment]
Yes.  The column label is MAG, and I would expect that column to be our best estimate of the observed mag.  I guess my only real concern would be downstream use of the starcat database entry where we might be collecting statistics comparing observed to catalog.  Even in those cases what I *really* care about is observed - expected.

**Comment by jeanconn** [comment]
Interesting.  OK.  I believe right now we do an agasc lookup for most of
the database tables instead of taking the MAG from the starcheck output,
but I could certainly handle both.

**Comment by jeanconn** [comment]
For the updated mag, if previous observations show a wide range, should we use the min, the mean, the more recent, or fall back to the catalog value.

**Comment by taldcroft** [comment]
Just use the median.  You can even include cases where it was not acquired (as 13.9 mag?).  This will bias the value upward which makes sense for non-acq cases.  Maybe the min N should be 7 instead of 5.

**Comment by jeanconn** [comment]
Does use as a BOT start count as one or two of the N here?

**Comment by taldcroft** [comment]
One.

**Comment by jeanconn** [comment]
I was hoping to just plug in this new median value in identify_stars(), but
it looks like we still need the AGASC magnitude for the MAXMAG check later
(ACA-041
https://github.com/sot/starcheck/blob/0b1ed55baddc7ad8f94b55072955616105226853/src/lib/Ska/Starcheck/Obsid.pm#L1312
).


​

**Comment by taldcroft** [comment]
We need to consult with Eric to remind ourselves how MAXMAG gets used on board.  Obviously ACA-041 was there for a reason, but it might not be the one we are imagining.

Anyway, I think we should put a moratorium on starcheck enhancements like this and focus on starhopper.

**Comment by jeanconn** [comment]
OK.  Not sure which issues are "like this" though.  I thought you wanted
the updated mag info for the best acquisition probabilities before this
next release.​

**Comment by jeanconn** [comment]
But I suppose you already have the actual acq histories in FigureOfMerit so
this would have done more with the overall warnings based on mag.
